REVIEW_PROCESS.md

```diff
@@ -0,0 +1,21 @@
+# Review Process Guidelines
+
+Refer to https://help.github.com/articles/about-pull-request-reviews/ for details on Pull Request review process at GitHub.
+
+For Host OS project, also consider the following guidelines:
+
+## Avoid stacked Pull Requests
+
+* Pull Requests (PR) should be independent of each other.
+* If any, identify dependencies in the very first line of the PR description.
+* GitHub does not allow approving only a sub-set of commits from a PR. When commits are updated but the reviewing commit is not, there is no way to distinguish this in the GitHub interface.
+
+## Avoid unrelated commits in the Pull Request
+
+* Non-related changes make difficult to review and deviate the original purpose of the PR.
+* Maintainers will close PR with unrelated changes and ask submitter to submit a new PR without the noise commits.
+
+## Pull Requests not passing tests are not approved
```

Comment:
As you wrote it, this is not a guideline. I suggest instead "Do not approve a Pull Request which does not pass tests (only for reviewers)"